src/world/config/item-ids.ts

```diff
@@ -52,4 +52,44 @@ export const itemIds = {
     runiteBar: 2363,
     runiteOre: 451,
     beer: 1917,
+    pureEssence: 7936,
+    runeEssence: 1436,
+    airTalisman: 1438,
```

Comment:
Would you consider putting these into a more organized nested format? Ie...
```
talismans: {
    air: ...
    water: ...
...
}
```

?